source/administration/console/security-and-access.rst

```diff
@@ -154,4 +156,4 @@ Configuring an external IDentity Provider (IDP) enables Single-Sign On (SSO) wor
 Use the the screens in this section to view, add, or edit an LDAP configuration for the deployment.
 MinIO only supports one active LDAP configuration.
 
-MinIO queries the active Active Directory / LDAP server to verify the credentials specified by the application and optionally return a list of groups in which the user has membership. 
\ No newline at end of file
+MinIO queries the active Active Directory / LDAP server to verify the credentials specified by the application and optionally return a list of groups in which the user has membership. 
```

Comment:
Hm. Rather than "Optionally", it's "if configured". But that is a hairy sentence.

```suggestion
MinIO queries the Active Directory / LDAP server to verify the client-specified credentials. 
MinIO also performs a group lookup on the AD/LDAP server if configured to do so.
```

Either way wasn't sure if "the active Active Directory.." was intentional. Probably a whoopsie from the before times.